lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -102,6 +102,35 @@ export default function LaunchErrorDialog({
           <p>Hypothesis needs your authorization to launch this assignment.</p>
         </BaseDialog>
       );
+    case 'blackboard_file_not_found_in_course':
+      return (
+        <BaseDialog
+          busy={busy}
+          error={error}
+          onRetry={onRetry}
+          title="Hypothesis couldn't find the file in the course"
+        >
+          <p>This might have happened because:</p>
+
+          <ul>
+            <li>
+              The file has been deleted from Blackboard: an instructor needs to
+              re-create the assignment with a new file
+            </li>
+            <li>
+              You don{"'"}t have permission to read the file: an instructor
+              needs to{' '}
+              <a
+                target="_blank"
+                rel="noreferrer"
```

Comment:
The use of `noreferrer` is consistent with the other cases in this module, but it would be good to do away with the need for this attribute in future, as it may be useful for support if they can track how much inbound traffic to knowledgebase articles comes from these help pages. We could perhaps also facilitate that with some suitable query string.